infrastructure/quick-deploy/aws/parameters.tfvars

```diff
@@ -359,6 +359,13 @@ node_exporter = {
   node_selector = {}
 }
 
+windows_exporter = {
+  node_selector = {
+    "service" = "windows"
```

Comment:
the label should most likely not be service but os or plateform.

Reply:
yes, platform will be used